services/quests/src/components/base/TextView.tsx

```diff
@@ -135,6 +136,41 @@ export default class TextView extends React.Component<TextViewProps, {}> {
 
       ref.editor.on('changeSelection', this.onSelectionChange);
       ref.editor.on('gutterclick', (e: any) => {this.onGutterClick(e); });
+      ref.editor.on('click', (e: any) => {
+        if (!e.domEvent.altKey) {
+          return;
+        }
+        ref.editor.commands.commands.jumpToDef.exec();
+      });
+
+      ref.editor.commands.addCommand({
+        name: 'jumpToDef',
+        hint: 'Jump to defintion',
+        bindKey: {win: 'Ctrl-I',  mac: 'Command-I'},
```

Comment:
Overall this looks great! Only thing I can think of would be to change this to ctrl/cmd - J for "jump to"

Reply:
I thought the same thing, but in Chrome ctrl-j is the shortcut to open the downloads page. I think Ace is supposed to stop those events from bubbling, but while I was testing it didn't seem to do so consistently. Seemed better to me to just go with something that isn't a conflict.

Reply:
Makes sense to me, thank you for clarifying!